README.md

```diff
@@ -1,18 +1,59 @@
 [comment]: # ( Copyright Contributors to the Open Cluster Management project )
 
-# Configuration Policy Controller [![KinD tests](https://github.com/open-cluster-management/config-policy-controller/actions/workflows/kind.yml/badge.svg?branch=main&event=push)](https://github.com/open-cluster-management/config-policy-controller/actions/workflows/kind.yml)
+# Configuration Policy Controller
+
 Red Hat Advanced Cluster Management - Governance - Configuration Policy Controller
 
-## How it works
+[![Build](https://img.shields.io/badge/build-Prow-informational)](https://prow.ci.openshift.org/?repo=open-cluster-management%2Fconfig-policy-controller)
+[![KinD tests](https://github.com/open-cluster-management/config-policy-controller/actions/workflows/kind.yml/badge.svg?branch=main&event=push)](https://github.com/open-cluster-management/config-policy-controller/actions/workflows/kind.yml)
+[![License](https://img.shields.io/:license-apache-blue.svg)](http://www.apache.org/licenses/LICENSE-2.0.html)
+
+## Description
 
-The Configuration Policy Controller watches for the following triggers to execute a reconcile:
+The Configuration Policy Controller watches `ConfigurationPolicies`, and checks if the specified objects are present in the cluster. It records compliancy details in the `status` of each ConfigurationPolicy, and as Kubernetes Events. If set to `enforce` the configuration, then the controller will attempt to create, update, or delete objects on the cluster as necessary to match the specified state. The controller can be run as a stand-alone program or as an integrated part of governing risk with Red Hat Advanced Cluster Management for Kubernetes.
+
+This is an example spec of a `ConfigurationPolicy` object:
+```yaml
+apiVersion: policy.open-cluster-management.io/v1
+kind: ConfigurationPolicy
+metadata:
+  name: policy-pod-example
+spec:
+  remediationAction: enforce
+  severity: low
+  namespaceSelector:
+    exclude: ["kube-*"]
+    include: ["default"]
+  object-templates:
+    - complianceType: musthave
+      objectDefinition:
+        apiVersion: v1
+        kind: Pod
+        metadata:
+          name: sample-nginx-pod
+        spec:
+          containers:
+          - image: nginx:1.18.0
+            name: nginx
+            ports:
+            - containerPort: 80
+```
 
-1. ConfigurationPolicy changes in all watched namespaces on the hub cluster
+The `ConfigurationPolicy` spec includes the following fields:
 
-Every reconcile the controller will:
+| Field | Description |
+| ---- | ---- |
+| severity | Optional: `low`, `medium`, or `high`. |
+| remediationAction | Required:  `inform` or `enforce`. Determines what actions the controller will take if the actual state of the object-templates does not match what is desired. |
+| namespaceSelector | Required: an object with `include` and `exclude` lists, specifying where the controller will look for the actual state of the object-templates. |
```

Comment:
Not really required. This field can be used if the objects are in multiple namespaces. If a namespace is defined in Kubernetes object from `objectDefinition`, `namespaceSelector` will be ignored. It will also be ignored, if the resource is cluster scoped

Reply:
Good point, if a bit unexpected (for a new user). That means that I could "exclude" the namespace that I put in the `objectDefinition` and the object would be created.

Also thanks for noting this; now I see the logic for the cluster-scoped resources, and that's pretty cool.